templates/csharp/copilot-plugin-from-scratch/{{ProjectName}}.csproj.tpl

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>{{TargetFramework}}</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <OutputType>Exe</OutputType>
```

Comment:
Does this strict that this project can only be run in Windows OS. What if we don't add this configuration?

Reply:
No, this does not mean that the function can only run on Windows OS, this indicates that we want to generate a .NET console app executable running in a separate process. If we remove this configuration, it will be a `.NET library` that can be used as a reference in other projects.